src/dataModels/JSON.js

```diff
@@ -721,50 +725,73 @@ var JSON = DataModel.extend('dataModels.JSON', {
  * @this {dataModels.JSON}
  * @memberOf dataModels.JSON~
  */
-function getUnderlyingIndexesOfSelectedRows() {
-    var sourceIndexes = [],
```

Comment:
Is there a deprecation warning for this?

Reply:
"Inner" function (not exposed)

Reply:
oh sorry, I should have expanded the view